app/utils.py

```diff
@@ -104,14 +135,16 @@ def gen_data(ckan, pids):
             status = 'Invalid frequency. Status could not be determined.'
 
         data = {
-            'dataset_id': pid,
+            'dataset_id': package['id'],
             'dataset_name': package['name'],
+            'dataset_title': package['title'],
             'last_updated': str(last_updated),
             'needs_update': status in statuses[1:],
             'status': status,
-            'age': age.days,
+            'age': int(age.days),
```

Comment:
For some reason it was returning a float (i.e. `4.0`). That patch seemed to fix it.